### DISCUSSION TITLE
And...Action! [GitHub Actions check-in] ▶️

### DISCUSSION THREAD

**Discussion by grandmas-favorite** [community/community#103166] [Actions]
![Header for the Actions category shows a dark purple abstract background with white text reading “GitHub Actions Check-in”. In the foreground there is an image of a person holding a magnifying glass next to a cartoon representation of a computer screen with a moving gear and play button. To the right, there is a GitHub logo in two-tone purple.](https://github.com/community/community/assets/144172738/72a8f806-9de3-409d-99e2-8e579a692491)

Hello Community! 👋 Hard to believe it’s already February, but the turn of the month means we are back with another Actions check-in!

2024 got off to a great start in January, and it’s all thanks to our wonderful community citizens who deserve recognition for starting the new year off right. 🚀 

Take a look below at some of the key contributors from last month:

- [CybCom](https://github.com/CybCom) needed to know how to fix a CI workflow without disturbing their main branch, and [faridnec](https://github.com/faridnec) came to the rescue with this [concise and elegant answer](https://github.com/orgs/community/discussions/83572) Hats off to [faridnec](https://github.com/faridnec)! 👏

- Shout out to [KK-Repos](https://github.com/KK-Repos) for helping [sr-murthy](https://github.com/sr-murthy) figure out how to [run a job on different runners based on the triggering branch](https://github.com/orgs/community/discussions/88552), creating a solution that other members can check out when they need help! 🤝

- We love a good self-solve because it means someone came back to the Community to share their solution with everyone else. Thanks to [rizwannadeem2017](https://github.com/rizwannadeem2017), anyone can learn how to [trigger an action on pull request based on the name of a branch](https://github.com/orgs/community/discussions/84990) 🏆

If you love open source like we do, you can always [contribute](https://github.com/github/docs/edit/main/content/index.md) directly to any doc by submitting a PR, learn more about [how to contribute](https://github.com/github/docs/blob/main/CONTRIBUTING.md) to GitHub docs. 📝 

### Changelog updates - What’s the latest in Actions? 🆕

- [The macOS 14 runner image is now available for GitHub hosted runners](https://github.blog/changelog/2024-01-30-github-actions-macos-14-sonoma-is-now-available/) 🍎 To use the runner, simply update the runs-on: key in your YAML workflow file to macos-14, macos-14-xlarge, or macos-14-large. Furthermore, the [M1 macOS runner](https://github.blog/changelog/2024-01-30-github-actions-introducing-the-new-m1-macos-runner-available-to-open-source/) is also now available to open source and is available for all plans, free in public repositories, and eligible to consume included free plan minutes in private repositories!  🎉

- I pinned this update to the Actions category earlier this month, but for anyone who missed it, we’re excited to announce that [The Repository Actions Runners List is now generally available](https://github.blog/changelog/2024-01-17-github-actions-repository-actions-runners-list-is-now-generally-available/), meaning that you can now view all available runners right within the Actions tab, without needing access to repository or organization settings. ▶️

### Some tips & tricks to level up your learning game 📈

-  **Show everyone that you’re a GitHub expert** with our [GitHub Certifications](https://github.blog/2024-01-08-github-certifications-are-generally-available/), now generally available. What was once reserved for employees and partners is now available to anyone who wants to prove their GitHub wit! 📜

- **Looking for Github mobile tips?**  Check out our latest edition of our [Insider newsletter digest: Build from anywhere](https://github.blog/2024-01-18-insider-newsletter-digest-build-from-anywhere/), where we discuss how to use GitHub mobile’s issue composer! 🤳

- **How does your work affect someone across the world?** 🌍 If you’re curious about this, check out our recent blog post about the [GitHub Innovation Graph](https://github.blog/2024-01-18-new-data-and-visualizations-highlight-the-resilience-of-international-developer-collaboration/), where we release quarterly data on eight metrics–Git pushes, repositories, developers, organizations, programming languages, licenses, topics, and economy collaborators. 🗺️

What are you most excited about for the month of February?

**Comment by BigDudeCameo** [comment]
What do I do now if I messed up?

**Comment by a2425rdl** [comment]
up
